### PR TITLE
DEP-142: Fix instadeploy

### PR DESCRIPTION
**Pull request by joshp123**
Also reverts the revert of #112 - the python 3 changes.

Use rebase and merge to ensure the history is clean.

https://mendix.atlassian.net/browse/DEP-142

### CODE REVIEW

**Comment by joshp123** [comment]
Yeah, I figured it would be easier to just use GitHub's rebase and merge, but it's probably better to rebase it and merge as normal. Done.

**Comment by jtwaleson** [comment]
Better :) I think we can rebase it to just 1 commit?

**Comment by joshp123** [comment]
Also done!

File: lib/instadeploy.py [hunk 1]
```diff
@@ -115,7 +130,8 @@ def process_request(self):
             logger.warning('InstaDeploy terminating with MxBuildFailure: {}'.format(mbf.message))
             return (200, {'state': 'FAILED'}, mbf.mxbuild_response)
 
-        except Exception:
+        except Exception as e:
```
Comment (jtwaleson) [inline lib/instadeploy.py]: Unused variable: `as e`

**Comment by jtwaleson** [comment]
Ok, good to go!

**Pull request merged**